modules/web/js/ballerina/views/ballerina-file-editor.js

```diff
@@ -767,6 +775,13 @@ define(['lodash', 'jquery', 'log', './ballerina-view', './service-definition-vie
             // adding current package to the tool palette with functions, connectors, actions etc. of the current package
             this.addCurrentPackageToToolPalette();
             this._model.accept(this);
+
+            // adding declared import packages to tool palette
+            _.forEach(this._model.getImportDeclarations(), function (importDeclaration) {
+                var package = BallerinaEnvironment.searchPackage(importDeclaration.getPackageName());
+                self.toolPalette.getItemProvider().addImport(package[0]);
```

Comment:
Yeah, it could have been null. But now the tool pallet is rendered declaratively using react. So all the imports in the ast root node is added rendered in the tool pallet. The problem does not occur anymore.